projects/plugins/jetpack/sal/class.json-api-metadata.php

```diff
@@ -53,8 +53,14 @@ public static function is_internal_only( $key ) {
 			return false;
 		}
 
-		// We want to always return the `_jetpack_newsletter_access` key to display the correct newsletter access in Calypso.
-		if ( $key === '_jetpack_newsletter_access' ) {
+		// We want to always return the `_jetpack_newsletter_access` key to
+		// display the correct newsletter access in Calypso.
+		$whitelist = array(
+			'_jetpack_newsletter_access',
```

Comment:
Do you think we can use META_NAME_FOR_POST_TIER_ID_SETTINGS and META_NAME_FOR_POST_LEVEL_ACCESS_SETTINGS? Just so we don't miss them in the future? I am not sure about the env context...

Reply:
I'm not sure, I thought there must have been a reason, but I don't see a separate commit in the repo that tried it. I'm a little leery of making the change to both constants.  Should we just try with `META_NAME_FOR_POST_TIER_ID_SETTINGS`? (I'm checking in that change)